babel/messages/pofile.py

```diff
@@ -309,6 +323,10 @@ def _process_string_continuation_line(self, line, lineno) -> None:
         else:
             self._invalid_pofile(line, lineno, "Got line starting with \" but not in msgid, msgstr or msgctxt")
             return
+        assert line[0] == '"'
+        if line[-1] != '"':
+            self._invalid_pofile(line, lineno, "String must be delimited by double quotes")
```

Comment:
Would it be useful to note that this is a continuation context? Also, this message wouldn't be shown if the above assertion raises (unless asserts have been disabled with `-O`...), so maybe the message could be more specific?
```suggestion
            self._invalid_pofile(line, lineno, "Continuation string must end with double quotes")
```

Reply:
Happy to change the message. I added the assert because it's an invariant (this function is allied only if the string starts with double quotes), and thus we don't need to perform the check on the first character, only the last.